include/Gaffer/MonitorAlgo.h

```diff
@@ -65,6 +65,7 @@ enum PerformanceMetric
 
 std::string formatStatistics( const PerformanceMonitor &monitor, size_t maxLinesPerMetric = 50 );
 std::string formatStatistics( const PerformanceMonitor &monitor, PerformanceMetric metric, size_t maxLines = 50 );
+std::string formatTotalStatistics( const PerformanceMonitor &monitor );
```

Comment:
Any reason this is a separate function rather than just being functionality of `formatStatistics()`? I can't think of a reason to not want the summary, and if I wanted only the summary, perhaps I could pass `maxLinesPerMetric = 0`. But if you do think it's important to keep it, then perhaps calling it `formatCombinedStatistics()` would make it more obvious that it outputs `monitor.combinedStatistics()`?